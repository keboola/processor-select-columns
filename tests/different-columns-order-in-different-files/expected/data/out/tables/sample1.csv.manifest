{"delimiter":",","enclosure":"\"","columns":["col_name_1","col_name_3"]}
