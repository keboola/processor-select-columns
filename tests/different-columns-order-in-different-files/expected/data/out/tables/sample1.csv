"col_name_1","col_name_3"
"col_val_1_1","col_val_1_3"
"col_val_2_1","col_val_2_3"
